actionpack/lib/action_dispatch/http/mime_types.rb

```diff
@@ -10,15 +10,34 @@
 Mime::Type.register "text/calendar", :ics
 Mime::Type.register "text/csv", :csv
 Mime::Type.register "text/vcard", :vcf
+Mime::Type.register "text/vtt", :vtt, %w(vtt)
 
 Mime::Type.register "image/png", :png, [], %w(png)
 Mime::Type.register "image/jpeg", :jpeg, [], %w(jpg jpeg jpe pjpeg)
 Mime::Type.register "image/gif", :gif, [], %w(gif)
 Mime::Type.register "image/bmp", :bmp, [], %w(bmp)
 Mime::Type.register "image/tiff", :tiff, [], %w(tif tiff)
 Mime::Type.register "image/svg+xml", :svg
+Mime::Type.register "image/webp", :webp, [], %w(webp)
+
+Mime::Type.register "audio/mpeg", :mpeg, [], %w(mp1 mp2 mp3)
```

Comment:
I'm curious. Is it safe for the same value to be registered in mine type symbols? This seems to cause warning in the definition of mime method. 
https://travis-ci.org/rails/rails/jobs/308298401#L1970

Reply:
That's right!! I'll remove the `video/mpeg` one since it contains old formats.